tfhe/src/shortint/server_key/mod.rs

```diff
@@ -745,21 +745,21 @@ impl ServerKey {
     /// # Example
     ///
     /// ```rust
-    /// use tfhe::shortint::gen_keys;
     /// use tfhe::shortint::parameters::{PARAM_MESSAGE_2_CARRY_2, PARAM_SMALL_MESSAGE_2_CARRY_2};
+    /// use tfhe::shortint::{gen_keys, CiphertextBig};
     ///
     /// // Generate the client key and the server key:
     /// let (cks, sks) = gen_keys(PARAM_MESSAGE_2_CARRY_2);
     ///
     /// let msg = 1;
     ///
     /// // Trivial encryption
-    /// let ct1 = sks.create_trivial(msg);
+    /// let ct1: CiphertextBig = sks.create_trivial(msg);
     ///
     /// let ct_res = cks.decrypt(&ct1);
     /// assert_eq!(1, ct_res);
     /// ```
-    pub fn create_trivial(&self, value: u64) -> CiphertextBig {
+    pub fn create_trivial<PBSOrder: PBSOrderMarker>(&self, value: u64) -> CiphertextBase<PBSOrder> {
         ShortintEngine::with_thread_local_mut(|engine| engine.create_trivial(self, value).unwrap())
     }
```

Comment:
if make this one generic then I guess we can remove the small from below ? Or am I missing something ?

What's the issue you were facing with this one ?

Reply:
I removed the create_trivial_small as its not needed

Reply:
only annoying thing is the need to annotate the type after the create trivial potentially if it cannot be deduced from context, this should be fine probably

Reply:
SHould be fine mostly, as you rarely only use trivial ciphertexts